app/views/play/CampaignView.coco.coffee

```diff
@@ -497,21 +497,20 @@ module.exports = class CampaignView extends RootView
       topScore = _.first(LevelSession.getTopScores({session: session.toJSON(), level}))
       @levelScoreMap[levelOriginal] = topScore
 
-  # Minecraft Modal:
-  maybeShowMinecraftModal: ->
+  userQualifiesForMinecraftModal: ->
     return false if me.freeOnly()
-    userQualifiesForMinecraftModal = (user) ->
-      return true if user.isAdmin()
-      return false if user.isPremium()
-      return false if user.isAnonymous()
-      return user.get('testGroupNumber') % 5 is 1
-
-    return unless userQualifiesForMinecraftModal(me)
-    if @campaign and @campaign.get('levels')
-      levels = @campaign.get('levels')
-      level = _.find(levels, {slug: "the-second-kithmaze"})
-      if level and @levelStatusMap['the-second-kithmaze'] is 'complete' and /^en/i.test(me.get('preferredLanguage', true))
-        $(".cube-level").show()
+    return false if storage.load 'roblox-clicked'
+    return true if me.isPremium()
+    return false if me.get('hourOfCode')
+    return false if me.isAnonymous()
```

Comment:
@adamkecskes We should show it to anonymous players after they have seen the paywall the first time.